test/compress/harmony.js

```diff
@@ -183,8 +183,9 @@ import_statement: {
         import { Bar, Baz } from 'lel';
         import Bar, { Foo } from 'lel';
         import { Bar as kex, Baz as food } from 'lel';
+        import * from 'lel';
     }
-    expect_exact: "import\"mod-name\";import Foo from\"bar\";import{Bar,Baz}from\"lel\";import Bar,{Foo}from\"lel\";import{Bar as kex,Baz as food}from\"lel\";"
+    expect_exact: "import\"mod-name\";import Foo from\"bar\";import{Bar,Baz}from\"lel\";import Bar,{Foo}from\"lel\";import{Bar as kex,Baz as food}from\"lel\";import*from\"lel\";"
```

Comment:
Please single quote this string so the double quotes are not escaped.

Better yet, use the array form of `expect_exact` - [example](https://github.com/mishoo/UglifyJS2/blob/7cb1adf455f8ab440e1971ae41265c1f7f9a806a/test/compress/loops.js#L397-L409)